new-log-viewer/src/services/LogFileManager/index.ts

```diff
@@ -226,39 +249,36 @@ class LogFileManager {
      * Gets the data that corresponds to the cursor.
      *
      * @param cursor
-     * @return Log event numbers for:
+     * @param numEvents
+     * @return Index for:
      * - the range [begin, end) of the page containing the matching log event.
      * - the log event number that matches the cursor.
      * @throws {Error} if the type of cursor is not supported.
      */
-    #getCursorData (cursor: CursorType): {
-        pageBeginLogEventNum: number,
-        pageEndLogEventNum: number,
-        matchingLogEventNum: number
+    #getCursorData (cursor: CursorType, numEvents: number): {
```

Comment:
Shall we extract a type `CursorData` from the return type and also replace the reference in `getLastEventCursorData()`?